resources/get-unmapped-dimension-values.sparql

```diff
@@ -0,0 +1,16 @@
+PREFIX qb: <http://purl.org/linked-data/cube#>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX ui: <http://www.w3.org/ns/ui#>
+
+SELECT ?dim ?member ?label WHERE {
+  ?ds qb:structure ?struct .
+  ?struct a qb:DataStructureDefinition .
+  ?struct qb:component ?comp .
+  VALUES ?dim { ?refareadim ?refperioddim }
+  ?comp qb:dimension ?dim .
+  ?comp qb:codeList ?list .
+  ?list skos:member ?member .
+  OPTIONAL { ?member rdfs:label ?label . }
```

Comment:
The ?member label property can be other that the rdfs:label e.g. skos:prefLabel. This information should be red from the configuration `codelist-label`